custom/icds_reports/middleware.py

```diff
@@ -29,7 +29,7 @@
 # after split we get ['', 'a', 'domain', 'icds-dashboard-view-name']
 def is_path_in_audit_urls(request):
     path = getattr(request, 'path', '').split('/')
-    return len(path) >= 3 and path[3] in AUDIT_URLS
+    return len(path) > 3 and (path[3] in AUDIT_URLS or (len(path) > 4 and  path[4] in AUDIT_URLS))
```

Comment:
Will it be worth to separate this to length specific (for length 4 and length 5) conditions to avoid the edge case where lengh is 5 and path[3] happens to be in AUDIT_URLS (theoretically possible)

Reply:
Also, should update the above comment to reflect the change you are doing.

Reply:
While that is possible, i think its better to err on the side of more logging. The worst that can happen in that case is that we get a record of something we did not need a record of which is fine, and the flexibility makes it more likely for us to catch a url if it changes.